docs/integrators/deep-history-squad.md

```diff
@@ -103,9 +103,30 @@ Never attach a non-pruned database to a regular observer (i.e. that does not hav
 
 Archives supporting historical lookup are available to download from a Google Cloud Storage bucket.
 
-:::important
-Documentation in this section is preliminary and subject to change.
-:::
+In order to avoid unintentional downloads and promote careful fetching of archives, we've enabled [requester pays](https://cloud.google.com/storage/docs/requester-pays).
+
+### Requirements
+
+1. **Google Cloud Platform Account**: An account on Google Cloud Platform with billing enabled is required. Find out how you can manage your billing account and modify your project here:
+- https://cloud.google.com/billing/docs/how-to/manage-billing-account
+- https://cloud.google.com/billing/docs/how-to/modify-project
+
+2. **Google Cloud SDK**: The Google Cloud SDK includes the `gcloud` command-line tool, which you'll use to interact with Google Cloud Storage. You can download it from the [Google Cloud SDK webpage](https://cloud.google.com/sdk/docs/install).
```

Comment:
Perhaps, instead of "You can download it from the", can be "In order to install it, please follow".